services/app-api/serverless.yml

```diff
@@ -249,7 +227,7 @@ functions:
           method: post
           cors: true
           authorizer: aws_iam
-    timeout: 60 # zipping large amount of files hits default time out of 6 seconds
+    timeout: 30 # zipping large amount of files hits default time out of 6 seconds
```

Comment:
assuming you intended to do this change - is it just that esbuild is faster so this time can go down i'm curious

Reply:
Any Lambda attached to an API gateway has a 30s max timeout, so there are warnings in the logs about this. Just bumped it down to silent those.